hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/CustomTypeFactory.java

```diff
@@ -241,4 +247,17 @@ public static Schema createSchema(String typeName, List<FieldDescriptor> fields)
     public static HazelcastJsonValue createHazelcastJsonValue(String value) {
         return new HazelcastJsonValue(value);
     }
+
+    public static ConfigUpdateResult createConfigUpdateResult(List<ConfigNamespace> addedConfigs,
+                                                              List<ConfigNamespace> ignoredConfigs) {
+        return new ConfigUpdateResult(new HashSet<>(addedConfigs), new HashSet<>(ignoredConfigs));
+    }
+
+    public static ConfigNamespace createConfigNamespace(String configSection, String configName) {
```

Comment:
If you know configName here is not null, you chould also add a filter about enum having multiple occurences.

Reply:
Deleted this method since we decided in the [client-protocol PR](https://github.com/hazelcast/hazelcast-client-protocol/pull/412#discussion_r794440801) that the operation will be async (hence no need to serialize/deserialize the `ConfigUpdateResult` in the client protocol).